tox.ini

```diff
@@ -131,7 +131,7 @@ markers =
     doctest: doctest tests
     # Other markers
     timeout: tests that need a timeout time
-addopts = --doctest-modules --timeout=300
+addopts = --doctest-modules --timeout=1200
```

Comment:
The purpose of the timeout is to stop stalling tests; having a wider timeout will help prevent tests from failing when many test runs are occurring simultaneously.